spec/integration/put_content/content_with_a_previously_published_edition_spec.rb

```diff
@@ -42,14 +51,58 @@
     expect(edition.user_facing_version).to eq(6)
   end
 
-  it "copies over the first_published_at timestamp" do
+  it "sets first_published_at to the previously published version's value" do
     put "/v2/content/#{content_id}", params: payload.to_json
 
     edition = Edition.last
     expect(edition).to be_present
     expect(edition.document.content_id).to eq(content_id)
 
-    expect(edition.first_published_at.iso8601).to eq(first_published_at.iso8601)
+    expect(edition.first_published_at).to eq(first_published_at)
+  end
+
+  it "sets temporary_first_published_at to the previously published version's value" do
+    put "/v2/content/#{content_id}", params: payload.to_json
+
+    edition = Edition.last
+    expect(edition.temporary_first_published_at)
+      .to eq(temporary_first_published_at)
+  end
+
+  context "and update_type is minor" do
```

Comment:
I think this should probably be a "when update_type is minor" as it doesn't appear to already be nested in a context block.